app/backend/approaches/chatreadretrieveread.py

```diff
@@ -109,12 +109,12 @@ async def run_until_final_call(
 
         # STEP 1: Generate an optimized keyword search query based on the chat history and the last question
         messages = self.get_messages_from_history(
-            self.query_prompt_template,
-            self.chatgpt_model,
-            history,
-            user_query_request,
-            self.query_prompt_few_shots,
-            self.chatgpt_token_limit - len(user_query_request),
+            system_prompt=self.query_prompt_template,
```

Comment:
I added keyword args to make it clear what the args are, given how many there are.